_includes/collections/workshops.html

```diff
@@ -1,13 +1,73 @@
 <div class="row">
+  {% assign sorted = (site.workshops | sort: 'date') | reverse %} {% for
+  workshop in sorted limit:1 %}
+  <div class="col s12 m4" style="display:block">
+    <div class="card medium sticky-action">
+      <div
+        class="card-image waves-effect waves-block waves-light"
+        style="{{workshop.style}}"
+      >
+        <img
+          class="activator"
+          src="{{workshop.img}}"
+          style="
```

Comment:
Shift the inline css to a fixed .css file.

Reply:
@Neilblaze  so in which file should i shift this styling

Reply:
> Shift the inline css to a fixed .css file.

or is think it should be just removed because ...i tried so and found nothing being affected..

Reply:
if the above css does nothing then feel free to remove it, if not then try shifting this to `assets/css/<filename.css>`